src/LineChart.ts

```diff
@@ -48,7 +49,7 @@ module n3Charts {
       scope.$watch('options+data', () => {
         // Call the update event with a copy of the options
         // to avoid infinite digest loop
-        eventMgr.trigger('update', scope.datasets, angular.copy(scope.options), attributes);
+        eventMgr.trigger('update', scope.datasets, new Utils.Options(angular.copy(scope.options)), attributes);
```

Comment:
Yeah, I guess we can ignore it for now, and if needed we'll either pass it again or just give it to the options (I prefer the latter).